client/src/actions/ConfigurationsActions.ts

```diff
@@ -117,6 +119,31 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
     };
   }
 
+  saveAsTemplate(template: IDashboardConfig) {
+    
+    return (dispatcher: (result: { template: IDashboardConfig }) => void) => {
+      let script = this.objectToString(template);
+      
+      script = '/// <reference path="../../../client/@types/types.d.ts"/>\n' +
+              'import * as _ from \'lodash\';\n\n' +
+              'export const config: IDashboardConfig = /*return*/ ' + script;
```

Comment:
This is to indicate the transformation of this file from ts to js (using return {...} instead of export)